tests/helpers.py

```diff
@@ -12,11 +17,13 @@ def make_transfer_trace(
     to_address: str,
     token_address: str,
     amount: int,
+    action: dict = {},
+    subtraces: int = 0,
```

Comment:
rather than giving people the _option_ to pass these, better to make it _explicit_ that no one does and not have them as arguments. Just set the values directly where they're used inside the function.